### PR TITLE
TouchEvent support

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

File: src/DOM/Event/TouchEvent.purs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+foreign import length :: TouchList -> Int
+
+foreign import _item :: Int -> TouchList -> Nullable Touch
+
+item :: Int -> TouchList -> Maybe Touch
+item i = toMaybe <<< _item i
+
+eventToTouchEvent :: Event -> F TouchEvent
+eventToTouchEvent = readTouchEvent <<< toForeign
```
Comment (garyb) [inline src/DOM/Event/TouchEvent.purs]: We don't provide these upcasts for any of the other event types as part of this library, so don't really need this one either. `purescript-dom-classy` covers this sort of thing instead. :)

**Review by garyb — CHANGES_REQUESTED** [review]

File: src/DOM/Event/TouchEvent.purs (hunk 1 above)
Comment (jacereda) [inline src/DOM/Event/TouchEvent.purs]: I copied `eventToTouchEvent` from `MouseEvent`, also present in `ClipboardEvent`, `CompositionEvent`, `FocusEvent`, `KeyboardEvent` and `WheelEvent`.  Are those kept for backwards compatibility?

Comment (garyb) [inline src/DOM/Event/TouchEvent.purs]: Uhh ok, I guess I'm wrong! I didn't think we provided those. :smile:

**Comment by garyb** [comment]
Thanks!

**Pull request merged**